### DISCUSSION TITLE
Expanding the Advanced Rules for a popup

### DISCUSSION THREAD

**Discussion by nviitala** [elementor/elementor#24401] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The advanced rules is currently quite limited. Is there any plans to expand those?

For example, I can not limit the popup to a specific locale/language which is assigned to the user (I am usin Polylang), so the popup will show in every language version of the site.

There is a "Show from arriving from a specific url" but no option for "Show on a specific url", which would help us to check the domain or url path and hide the popup based on it. We could then set the popup visible only on domain.com but not on domain.es, or hide it on /es/, for example.

### Describe the solution you'd like

- Advanced rules tab could include a possibility to limit the popup to be visible on a certain locale only.
- Conditions tab could include a regex possibility with which it would be possible to match any part of the whole url.

### Describe alternatives you've considered

Maybe some kind of filter which a programmer could use to hide or show popups, while respecting the chosen triggers and advanced rules?

**Comment by steveindzine** [comment]
This would be really handy, for example if someone hasn't added to the cart you could have an exit intent popup to remind them to add to cart, but if they have added to cart it wouldn't show. Something based on cookies or local storage, or some kind of JavaScript to hook into it so that it can be disabled.
